### PR TITLE
support the new description format

### PR DESCRIPTION
**Pull request by ericbmerritt**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,11 +7,13 @@ HOMEPAGE:="https://github.com/afiniate/ouija"
...
 
-DESC:="Ouija is a path resolution library for ocaml"
+DESC_FILE:=$(CURDIR)/description
 
 BUILD_DEPS:=vrt
-DEPS:=core async async_unix async_shell cohttp cohttp.async atdgen sexplib cryptokit
+DEPS:=core async async_unix async_shell cohttp atdgen sexplib cryptokit
 
+all:
+	echo "${DESC}"
```
Comment (quiquepaz) [inline Makefile]: Is this `echo` intentional? Also, `DESC` is no longer declared (now only `DESC_FILE` is)

Comment (quiquepaz) [inline Makefile]: Even more, `DESC` is mandatory since it's used by `vrt.mk` in the `metadata` goal

**Comment by quiquepaz** [comment]
This PR depends on https://github.com/afiniate/vrt/pull/14/

File: Makefile (hunk 1 above)
Comment (ericbmerritt) [inline Makefile]: no. its a leftover. Will remove.

**Pull request merged**